classes/binconfig.oeclass

```diff
@@ -45,6 +66,15 @@ def do_install_binconfig_fixup(d):
         with open(filename, "w") as output_file:
             output_file.write(binconfig_file)
 
+        # create symlink
+        srcdir = os.path.join("..", "..", "machine", d.get("machine_bindir").lstrip("/"))
+        srcfile = os.path.join(srcdir, os.path.basename(filename))
+        dstdir = os.path.join(d.get("D"), d.get("stage_bindir").lstrip("/"))
+        dstlink = os.path.join(dstdir, os.path.basename(filename))
+        oelite.util.makedirs(dstdir)
+        print "symlinking %s to %s"%(dstlink, srcfile)
+        os.symlink(srcfile, dstlink)
```

Comment:
The same issue arises in line 30:
`FILES_${BINCONFIG_CROSS_PACKAGE} = "${stage_bindir}/*-config"`